Modules/_datetimemodule.c

```diff
@@ -637,17 +637,22 @@ check_date_args(int year, int month, int day)
 {
 
     if (year < MINYEAR || year > MAXYEAR) {
-        PyErr_Format(PyExc_ValueError, "year %i is out of range", year);
+        PyErr_Format(PyExc_ValueError,
+                     "year must be in %d..%d, but got %d",
+                     MINYEAR, MAXYEAR, year);
         return -1;
     }
     if (month < 1 || month > 12) {
-        PyErr_SetString(PyExc_ValueError,
-                        "month must be in 1..12");
+        PyErr_Format(PyExc_ValueError,
+                    "month must be in 1..12, but got %d",
+                    month);
```

Comment:
Nit: this fits well within 79 chars.
Moverover: fix vertical alignment.

```suggestion
                     "month must be in 1..12, but got %d", month);
```